plone/app/contenttypes/behaviors/richtext.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from Acquisition import aq_base
 from plone.app.contenttypes import _
+from plone.app.dexterity.textindexer import searchable
```

Comment:
```suggestion
from plone.app.dexterity import textindexer
```

Reply:
it is better to import textindexer here, so that we see later if we are using `textindexer.searchable()` or `utils.searchable()`